qa/L0_trt_plugin/test.sh

```diff
@@ -63,14 +153,15 @@ if [ "$SERVER_PID" == "0" ]; then
     exit 1
 fi
 
+rm -f $CLIENT_LOG
 set +e
-python $PLUGIN_TEST >$CLIENT_LOG 2>&1
+python $PLUGIN_TEST PluginModelTest.test_raw_fff_clip
```

Comment:
You seem to be missing `>>$CLIENT_LOG 2>&1` here.

Reply:
Fixed, thanks.